src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, useMemo } from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[];
+  slug: string | undefined;
+};
+
+export const PeopleTable: FC<Props> = ({ people, slug }) => {
+  const getParent = useMemo(() => {
```

Comment:
useMemo optional here, because searching through an array is not a very difficult task to use this useMemo hook for it